docs/design/snapshot.md

```diff
@@ -0,0 +1,229 @@
+Snapshots are read-only copies of volumes (once the snapshot is activated) that
+can be used for cloning new volumes from. Its is possible to create a snapshot
```

Comment:
The word from at the end of the first sentence is awkward and not needed.

"Its is" should be corrected.